atest/robot/libdoc/spec_library.robot

```diff
@@ -1,4 +1,5 @@
 *** Settings ***
+Library           OperatingSystem
```

Comment:
Ought to be already imported by the imported resource file.

Reply:
There's no single best practice but we have generally imported only resource files in our tests if those resource files anyway import libraries we need. One use for resource files with Robot is to avoid importing multiple libraries or resource files in a test case file to keep them simpler.

Reply:
I see!